### ISSUE TITLE
fnm node version not showing

### ISSUE THREAD

**Issue by MauricioRobayo** [IlanCosman/tide#224]
I was using [Volta](https://volta.sh/) for installing and switching between node versions, and I had the node version properly displayed on the right without any additional configurations, it just worked out of the box.

Now I switched to [fnm](https://github.com/Schniz/fnm) and the node version stopped showing on the right.

Is there something I need to enable to show the node version on the right while using [fnm](https://github.com/Schniz/fnm)?

**Comment by IlanCosman** [comment]
Do you still have `node` in your `tide_right_prompt_items`? 
i.e run: `contains node $tide_right_prompt_items && echo "node is here"`

Tide automatically removes the node item if you don't have `node` installed.

The node item also only displays when you there is a `package.json` in the current directory.

**Comment by MauricioRobayo** [comment]
Hi, I'm really sorry for the late response, I drowned in work past week.

I guess node is no longer on `tide_right_prompt_items`, there is no output when I run `contains node $tide_right_prompt_items && echo "node is here"`

But node is installed, if I run `node -v` I get the correct version.

I tried switching to several directories with package.json files but still not showing, even when there is a `.nvmrc` file I can see `fnm` is switching to the specified version of node as it displays a message like `Using Node v14.17.4` when moving into the directory, and node works fine with the correct version, but not showing on Tide's prompt.

Am I missing some configuration. I'll be glad to provide any additional info as needed.
